addons/test_discuss_full/tests/test_performance.py

```diff
@@ -1058,6 +1058,7 @@ def _expected_result_for_message(self, channel):
                 "thread": {"id": channel.id, "model": "discuss.channel"},
                 "parentMessage": False,
                 "pinned_at": False,
+                "rating_id": False,
```

Comment:
Note: adding missing keys as a good practice, as absence of field is not equal to empty field